### ISSUE TITLE
Package repository needs to be re-signed without SHA-1 for Debian 9 

### ISSUE THREAD

**Issue by kiboflavin** [holland-backup/holland#183]
I just upgraded from Debian 8 "jessie" to Debian 9 "stretch".  Upon doing an "apt-get upgrade" on my system, I get the following error from the holland repo:

`W: GPG error: http://download.opensuse.org/repositories/home:/holland-backup/Debian_8.0 ./ Release: The following signatures were invalid: 5B31400395D88EF2FCA55ECF40A402AF984D0514`

I did a bit of research, and it seems the Apt developers have removed support for GPG keys that include an SHA-1 digest as part of the key preferences.  Support for SHA-1 was removed as of APT version 1.4b1.  

I found this stackexchange post that explains the problem and how to fix it:  https://unix.stackexchange.com/questions/387053/debian-9-apt-and-gpg-error-inrelease-the-following-signatures-were-inva

TL;DR: The Release file in the apt repository needs to be re-signed with a key that has the SHA-1 digest disabled and re-published for Debian 9 users.  Probably Ubuntu users as well.

**Comment by xarr** [comment]
This is not meant as a hijack, but would also welcome updated Debian 9 (stretch) packages on:

* http://download.opensuse.org/repositories/home:/holland-backup/

Thanks!

**Comment by sebastianhelbig** [comment]
Anyone know a workaround?

**Comment by soulen3** [comment]
I've uploaded a new key and created a package for Holland 1.1.8
